### PR TITLE
ADEN-14545 :: Expanded commands and documentation

### PR DESCRIPTION
**Pull request by p-forfa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,6 +186,32 @@ or the simpler version:
...
+```bash
+app/console order:add-creatives -o ORDER_ID -c CREATIVE_TEMPLATE_ID --force-new-creative --multiple-creatives-per-line-item 2 --override-creative-size '1x1,300x250'
+```
+
+Going off of the above option of having multiple creatives per line item, and the previously mentioned suffix option, there's also an option to append an index value at the end of each creative's name.
+This option is `--append-loop-index`. It does not require another option to be present, but it makes the most sense to activate when the `--multiple-creatives-per-line-item` option is added. If you just activate it,
+and don't add in the former option or add it in with a value of 1, then the index will always be 1. This will effectively add a '(1)' at the end of your creative name.
+
+For example, you can run the following command with the :
+```bash
```
Comment (vforge) [inline README.md:214]: @p-forfa Missing command

File: README.md [hunk 2]
```diff
@@ -210,6 +268,8 @@ If the built image somehow does not get pushed to the k8s check the latest versi
 
 In order to get to the k8s dashboard you need to use [Valut](https://wikia-inc.atlassian.net/wiki/spaces/OPS/pages/132317429/Vault%2BFor%2BEngineers) to get the dashboard-user token.
 
+When adding a new command, with a new php file to the app/console file, make sure to run `composer install'. You may get some odd errors about the php file not being found, if you do not do this.
```
Comment (vforge) [inline README.md]: Incorrect backtick after `composer install` breaking formatting

File: README.md (hunk 1 above)
Comment (p-forfa) [reply README.md:214]: Good catch. Fixed in e172a9f43787395db4d65d70ae2824b7b9523ff1

File: README.md (hunk 2 above)
Comment (p-forfa) [reply README.md]: Fixed in b72c1c4427420d8cd67643b5d17e14e0419b8924

**Pull request merged**